performance_test.go

```diff
@@ -1,7 +1,7 @@
 package gubrak
 
 import (
-	"github.com/gocarina/gocsv"
+	"github.com/DefinitelyMod/gocsv"
```

Comment:
You might wonder why we use `gocsv` from `DefinitelyMod` instead of the original author.

There is an issue on the original repo https://github.com/gocarina/gocsv/issues/100 about Backward compatibility with go modules and the author just doesn't care.